test/e2e/helpers/test-utils.ts

```diff
@@ -57,7 +57,8 @@ export function getBpmnDiagramNames(directoryName: string): string[] {
     .map(filename => filename.split('.').slice(0, -1).join('.'));
 }
 
-export async function getContainerCenter(containerElement: ElementHandle<SVGElement | HTMLElement>): Promise<Point> {
+export async function getContainerCenter(): Promise<Point> {
+  const containerElement: ElementHandle<SVGElement | HTMLElement> = await page.waitForSelector('#bpmn-container');
```

Comment:
⚠️ I suggest we move `getContainerCenter` into the `PageTester` class.
The function now uses an hard coded bpmn-container id, so it is hard to understand where it comes from and the function will fail if used with a page using a different id.
As the function is always used in tests that already use a `PageTester` instance, I suggest we move the function into the class, so it will be able to use the container id defined for the page